lib/ex_plasma/payment_v1_builder.ex

```diff
@@ -0,0 +1,106 @@
+defmodule ExPlasma.PaymentV1Builder do
```

Comment:
Not sure how I feel about this. The intention was that `ExPlasma.Builder` would be able to craft transaction for any of the sub types but maybe that's not possible?

Reply:
Yeah it's a bit hard to make it clean generic. The main issue is that the fee transaction is quite different from the payment transaction. It requires a block number, which looks weird to add to a generic builder.

What I could do maybe is make a `PaymentBuilder` and a `FeeBuilder` because I'm more confident that different versions of payment transactions will look the same. Or I just leave it like this, WDYT?

Reply:
@mederic-p Let's leave it like this for now. Since it's not very clear, I don't want to spend too much effort on this section as this really is just a handy way to craft the transactions quickly(especially around Payment V1 transactions). We can revisit once we get clarity on additional types in the future.